app/models/app_meta_datum.rb

```diff
@@ -0,0 +1,25 @@
+class AppMetaDatum < ApplicationRecord
+  belongs_to :application_type
+
+  enum :operation => {"s3" => 0, "role" => 1, "policy" => 2, "bind_role" => 3}
+
+  def self.seed
+    transaction do
+      YAML.safe_load(File.read("db/seeds/superkey_metadata.yml"))
+          .deep_symbolize_keys
+          .each do |app, settings|
+        apptype = ApplicationType.find_by(:name => app)
+
+        settings[:steps].each do |step|
+          AppMetaDatum.create!(
```

Comment:
I don't mind the naming.  I'm okay with it if we don't expose it via the API - I'm concerned if we do that the naming will be slightly confusing ( even if the use of `Datum` is correct. ) But I'm not going to split hairs over it.

Reply:
I'm for rename, it's confusing to me

Reply:
So I'm all for changing the name as well, but running into an issue with the common gem here: https://github.com/RedHatInsights/insights-api-common-rails/blob/12078a13696efe07beddc3a97f173f7cdde47e5f/lib/insights/api/common/application_controller_mixins/api_doc.rb#L32

Basically we are calling `singularize` on everything which makes sense. Would we be ok with my overriding this method in the sources ApplicationController? Since this is a one-off. It's either that or use the funky name in the openapi but _not_ in the application.